tests/test_generator/test_lengths.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+
+from gutenTAG import GutenTAG
+
+
+class TestLengths(unittest.TestCase):
+    def setUp(self) -> None:
+        self.config = {
+            "timeseries": [
+                {
+                    "name": "test-amp",
+                    "length": 500,
+                    "base-oscillations": [{"kind": "sine"}],
+                    "anomalies": [
+                        {
+                            "position": "end",
+                            "length": 47,
+                            # "creeping-length": 12,  # remove to test the default 20% option
+                            "channel": 0,
+                            "kinds": [{"kind": "amplitude", "amplitude_factor": 2.0}],
+                        }
+                    ],
+                }
+            ]
+        }
+        self.creeping_factor = 0.3
+
+    def update_config(self, length: int, include_creeping: bool = False):
```

Comment:
Please indicate that this method is private with `_`-prefix:
```suggestion
    def _update_config(self, length: int, include_creeping: bool = False) -> Dict[str, Any]:
```